recipe/meta.yaml

```diff
@@ -13,17 +13,20 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/SimpleITK/SimpleITK/releases/download/v{{ version }}/SimpleITK-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+# temporary for testing
+  git_url: https://github.com/SimpleITK/SimpleITK
+  git_tag: v{{version}}
+#  url: https://github.com/SimpleITK/SimpleITK/releases/download/v{{ version }}/SimpleITK-{{ version }}.tar.gz
+#  sha256: {{ sha256 }}
```

Comment:
Any reason to keep the commented out lines?

Reply:
This is explained by the comment above: "temporary for testing"